integrations/RollBar/browser.js

```diff
@@ -0,0 +1,502 @@
+          if (
+            (o.r(n),
+            Object.defineProperty(n, "default", { enumerable: !0, value: r }),
+            2 & e && "string" != typeof r)
+          )
+            for (var t in r)
+              o.d(
+                n,
+                t,
+                function (e) {
+                  return r[e];
+                }.bind(null, t)
+              );
+          return n;
+        }),
+        (o.n = function (r) {
+          var e =
+            r && r.__esModule
+              ? function () {
+                  return r.default;
+                }
+              : function () {
+                  return r;
+                };
+          return o.d(e, "a", e), e;
+        }),
+        (o.o = function (r, e) {
+          return Object.prototype.hasOwnProperty.call(r, e);
+        }),
+        (o.p = ""),
+        o((o.s = 0));
+    })([
+      function (r, e, o) {
+        "use strict";
+        var n = o(1),
+          t = o(5);
+        (_rollbarConfig = _rollbarConfig || {}),
+          (_rollbarConfig.rollbarJsUrl =
+            _rollbarConfig.rollbarJsUrl ||
+            "https://cdn.rollbar.com/rollbarjs/refs/tags/v2.24.0/rollbar.min.js"),
+          (_rollbarConfig.async =
+            void 0 === _rollbarConfig.async || _rollbarConfig.async);
+        var a = n.setupShim(window, _rollbarConfig),
+          l = t(_rollbarConfig);
+        (window.rollbar = n.Rollbar),
+          a.loadFull(
+            window,
+            document,
+            !_rollbarConfig.async,
+            _rollbarConfig,
+            l
+          );
+      },
+      function (r, e, o) {
+        "use strict";
+        var n = o(2),
+          t = o(3);
+        function a(r) {
+          return function () {
+            try {
+              return r.apply(this, arguments);
+            } catch (r) {
+              try {
+                console.error("[Rollbar]: Internal error", r);
+              } catch (r) {}
+            }
+          };
+        }
+        var l = 0;
+        function i(r, e) {
+          (this.options = r), (this._rollbarOldOnError = null);
+          var o = l++;
+          (this.shimId = function () {
+            return o;
+          }),
+            "undefined" != typeof window &&
+              window._rollbarShims &&
+              (window._rollbarShims[o] = { handler: e, messages: [] });
+        }
+        var s = o(4),
+          d = function (r, e) {
+            return new i(r, e);
+          },
+          c = function (r) {
+            return new s(d, r);
+          };
+        function u(r) {
+          return a(function () {
+            var e = this,
+              o = Array.prototype.slice.call(arguments, 0),
+              n = { shim: e, method: r, args: o, ts: new Date() };
+            window._rollbarShims[this.shimId()].messages.push(n);
+          });
+        }
+        (i.prototype.loadFull = function (r, e, o, n, t) {
+          var l = !1,
+            i = e.createElement("script"),
```

Comment:
@saikumarrs  `i.dataset.loader = LOAD_ORIGIN;` we can add it here as well right?